articles/batch/virtual-file-mount.md

```diff
@@ -434,6 +434,8 @@ new PoolAddParameter
     }
 }
 ```
+> [!NOTE]
+> Looking for examples of alternate scripts no #C-based? check here in [Mount Azure File to Azure Batch Pool](https://techcommunity.microsoft.com/t5/azure-paas-blog/mount-azure-file-share-to-azure-batch-pool-via-azure-powershell/ba-p/2243992)
```

Comment:
The bigger question is what are our guidelines about linking from docs to blogs?  
It's a really useful blog.   But does linking to it imply that the Blog is now official documentation?    If so, then I don't think we should link it...